information_sampling_task/experiment.js

```diff
@@ -153,9 +153,10 @@ var resetRound = function() {
 	colors = jsPsych.randomization.shuffle(['green', 'red', 'blue', 'teal', 'yellow', 'orange',
 		'purple', 'brown'
 	]).slice(0,2)
-	numbersArray = jsPsych.randomization.repeat(numbers, 1)
-	color1_index = numbersArray.slice(0,13)
-	color2_index = numbersArray.slice(13)
+	var numbersArray = jsPsych.randomization.repeat(numbers, 1)
+	var num_majority = Math.floor(Math.random()*5) + 13
```

Comment:
I wanted to constrain it to 13-18. So one array (color index1) will be
length 13-18 while the other is 25-that length.

On Friday, June 10, 2016, Vanessa Sochat <notifications@github.com> wrote:

> In information_sampling_task/experiment.js
> <https://github.com/expfactory/expfactory-experiments/pull/464#discussion_r66699807>
> :
>
> > @@ -153,9 +153,10 @@ var resetRound = function() {
> >  	colors = jsPsych.randomization.shuffle(['green', 'red', 'blue', 'teal', 'yellow', 'orange',
> >  		'purple', 'brown'
> >  	]).slice(0,2)
> > -	numbersArray = jsPsych.randomization.repeat(numbers, 1)
> > -	color1_index = numbersArray.slice(0,13)
> > -	color2_index = numbersArray.slice(13)
> > +	var numbersArray = jsPsych.randomization.repeat(numbers, 1)
> > +	var num_majority = Math.floor(Math.random()*5) + 13
>
> Do you want this to sample from the entire array of numbers? The max it
> will return is 18, and the numbers array has length 25.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/expfactory/expfactory-experiments/pull/464/files/25a8d2e0bb59aa9cef471c7a7b4d4c6723696db0#r66699807>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI5LgDfzOMbJa0h1Zhdtu7j1U3DKnHgrks5qKiXsgaJpZM4IzcXl>
> .
>


-- 
Best,
Ian